sdk/rust/src/test.rs

```diff
@@ -63,7 +53,7 @@ mod tests {
         let decodedata = &encode(PLAINT_TEXT).to_string()[..];
         let keyspecs = vec!["EH_AES_GCM_128", "EH_AES_GCM_192", "EH_AES_GCM_256", "EH_SM4_CTR"];
         for keyspec in keyspecs.iter() {
-            let keyid = client.create_key(keyspec, "EH_INTERNAL_KEY",None, None, None)
+            let keyid = client.create_key(keyspec, "EH_INTERNAL_KEY", "EH_KEYUSAGE_ENCRYPT_DECRYPT")
                         .await
```

Comment:
please add a negative case:  for symmetric key input a EH_KEYUSAGE_SIGN_VERIFY which expect failed